admin/load.php

```diff
@@ -336,3 +336,28 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Enable all non-experimental modules on plugin activation.
+ *
+ * @since 1.0.0
+ */
+function perflab_activation_hook() {
+	// Bail if option is already set with any value.
+	if ( get_option( PERFLAB_MODULES_SETTING, false ) ) {
+		return;
+	}
```

Comment:
I think this needs to be reverted to what it previously was: We should not modify the option if it's already set. Without the `false !==` bit, the code here would still modify the option if it was set to an empty array, for example if the user had deactivated all modules. In that case, they intentionally did that, so we shouldn't change it.

The goal here is to only set the option to all available (non-experimental) modules initially, e.g. when the plugin is _first_ activated and nothing is configured yet.
```suggestion
	// Bail if option is already set with any value.
	if ( false !== get_option( PERFLAB_MODULES_SETTING, false ) ) {
		return;
	}
```